lib/caesar.rb

```diff
@@ -1,6 +1,28 @@
 class Caesar
-  attr_reader :key
-
+  attr_reader :alphabet, :cipher
+  attr_accessor :shift
   def initialize
+    @alphabet = ('a'..'z').to_a.join + ('A'..'Z').to_a.join
+    @shift = 0
+  end
+
+  def to_cipher(string, shift)
+    @shift = shift
```

Comment:
The reason shift shouldn't be an instance variable is apparent in this method. We use the shift that is passed to this method. Shift is only relevant through this method call, not through the object's lifetime.